packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -1106,4 +1106,67 @@
       <protected>false</protected>
       <shortDescription>Opens in the browser tab</shortDescription>
     </labels>
+    <labels>
+        <fullName>quantic_ComponentError</fullName>
+        <value>{{0}} component error</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>c-quantic-pager component error</shortDescription>
+    </labels>
+    <labels>
+      <fullName>quantic_LookAtDeveloperConsole</fullName>
+      <value>Look at the developer console for more information.</value>
+      <language>en_US</language>
+      <protected>false</protected>
+      <shortDescription>Look at the developer console</shortDescription>
+    </labels>
+        <labels>
+      <fullName>quantic_InvalidMaxNumberOfDocumentSuggestions</fullName>
+      <value>"{{0}}" is an invalid maximum number of document suggestions. An integer greater than 0 was expected.</value>
+      <language>en_US</language>
+      <protected>false</protected>
+      <shortDescription>invalid maximum number of document suggestions.</shortDescription>
+    </labels>
+    <labels>
+      <fullName>quantic_InvalidNumberOfAutoOpenedDocuments</fullName>
+      <value>"{{0}}" is an invalid number of automatically opened document suggestions. A positive integer was expected.</value>
```

Comment:
@mmitiche Same idea here:

The number of automatically opened document suggestions must be greater that 0.